person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_survive
+    age >= 18 || parent_permission == true
+  end
```

Comment:
- [ ] great job on setting up the Person class 👍  however in the `can_use_survive` method you are supposed to call the private method 'of_age' like below

```
def can_use_survive
  of_age() || @parent_permission
end
```
notice how you don't have to add `== true` after @parent_permission as this is already a boolean value.
- [ ] secondly I would like to advise you to take notice of the typo you made in the method name, this might cause issues later on when using this method.
- [ ] and lastly I would like to bring to your attention that you should be refferring to the instance variable of `parent_permission` which is `@parent_permission`

Reply:
Properly addressed